server/index.js

```diff
@@ -1,10 +1,40 @@
 const express = require("express");
-const serverConfig = require("./config");
 const path = require("path");
+const bodyParser = require("body-parser");
+const mongoose = require("mongoose");
 
+//Initialize Express App
 const app = express();
 
+//Local Imports
+const serverConfig = require("./config");
+const posts = require('./routes/post.routes');
```

Comment:
Maybe we can have a single require for routes in general? 🙂 

Have an `index.js` in the the `/routes` folder which aggregates all the routes and in `server/index.js` we can just `require('/routes);` and use it across the server file?

For example
```js
const routes = require('./routes');

...

app.use('/api/' , routes.post);
```